### DISCUSSION TITLE
[QUESTION] Is there a way to recover a backup without installing longhorn aka k3s

### DISCUSSION THREAD

**Discussion by iceHtwoO** [longhorn/longhorn#5871] [General]
## Question
Hey, a year ago i've been using k3s with longhorn but have decided to switch back to a basic docker system because this setup was overkill for my homelab. Now I have found some longhorn backups on my NAS that I would love to convert back to normal usable files without having to install k3s and longhorn again. Help would be appreciated.

**Comment by derekbit** [comment]
You can check the [doc](https://longhorn.io/docs/1.4.1/advanced-resources/data-recovery/recover-without-system/).
